curves/jq255s/src/fields/tests.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Probably a good idea to either delete this, or add tests like the other curve crates